sysutils/sketchybar/Portfile

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               makefile 1.0
+PortGroup               xcodeversion 1.0
+
+github.setup            FelixKratz SketchyBar 2.11.1 v
+
+categories              sysutils
+platforms               darwin
```

Comment:
`platforms darwin` is now implicit, so you can remove this line

Reply:
fixed